draft-ietf-privacypass-architecture.md

```diff
@@ -199,6 +193,44 @@ does not reply to the Origin's challenge with a new request.
 ~~~
 {: #fig-overview title="Privacy pass redemption and issuance protocol interaction"}
 
+The end-to-end flow for Privacy Pass involves three different types of contexts:
+
+- Redemption context: The interactions and set of information shared
+between the Client and Origin.
+- Issuance context: The interactions and set of information shared
+between the Client, Attester, and Issuer.
+- Attestation context: The interactions and set of information shared between
+the Client and Attester only, for the purposes of attesting the vailidity of the Client.
+
+The privacy goals of Privacy Pass are oriented around unlinkability based on these
+contexts. In particular, Privacy Pass aims to achieve Origin-Client unlinkability. This
+means that given two redemption contexts, the Origin cannot determine if both
+redemption contexts correspond to the same Client or two different Clients.
+Informally, this means that a Client in a redemption context is indistinguishable
+from any other Client that might use the same redemption context. The set of Clients
+that share the same redemption (or issuance) context is referred to as an anonymity set.
+Privacy Pass also aims to achieve Issuer-Client unlinkability, which is similar to
+Origin-Client unlinkability in that a Client in an issuer context is indistingusihable
+from any other Client that might use the same issuer context.
+Depending on the deployment model, Privacy Pass might also aim to achieve Attester-Origin
+unlinkability. Similar to Origin-Client and Issuer-Client unlinkability, this means that given two attestation
+contexts, the Attester cannot determine if both contexts correspond to the same
+Origin or two different Origins. The set of Clients that share the same attestation
+context is referred to as an anonymity set.
+
+At a high level, these properties ensure that no single party amongst the Attester,
+Issuer, or Origin can link client identifying information to client activity, e.g.,
+the origin being accessed.
+
+The manner in which Origin-Client, Issuer-Client, and Attester-Origin unlinkability are achieved depends on the deployment
+model, type of attestation, and issuance protocol details. For example, as discussed
+in {{deployment}}, failure to use a privacy-enhancing proxy system such as Tor when
```

Comment:
I think this is now the earliest mention of Tor in the document? If so, we should move the reference here.